docker-for-aws/persistent-data-volumes.md

```diff
@@ -215,13 +209,11 @@ you already have too many EBS volumes or want to reduce the amount of time it
 takes to transfer volume data across availability zones.
 
 ```bash
-{% raw %}
```

Comment:
You do need these because of the double curly braces below, which will be interpreted as Liquid.